src/pages/cardsPage/CardsPage.jsx

```diff
@@ -0,0 +1,59 @@
+import { React, useState, useEffect } from 'react';
+import { Box, TextField, ThemeProvider } from '@mui/material';
+import { UserCard } from '../../components/UserCard/UserCard';
+import theme from './theme';
+import './CardsPageStyles.css';
+import fetchUserData from '../../api/usersApi';
+import useLocalStorage from '../../customHooks/useLocalStorage';
+
+function CardsPage() {
+
+  const [users, setUsers] = useState([]);
+  const [search, setSearch] = useState('');
+  const [favorites, setFavorites] = useLocalStorage([]);
+
+  useEffect(() => {
+    fetchUserData().then(fetchedData => setUsers(fetchedData));
+  }, []);
+
+  const handleInputChange = (event) => {
+    setSearch(event.target.value);
+  }
+
+  const filterUsers = (users, search) => {
+    return users.filter((user) => {
+      if (users.length !== 0 && search === '') {
+        return user;
+      } else if (user.name.toLowerCase().includes(search.toLowerCase())) {
+        return user;
+      } else {
+        return null;
+      }
+    });
+  }
+
+  return (
+    <ThemeProvider theme={theme}>
+      <Box className="CardsPage">
```

Comment:
Be consistent with the way you are describing classes. 
Either use styles (js) in all of the project or just import the css, but do only one of the above don't mix it. The key is **consistency**!! 
Also be consistent with the naming if you are using `camelCase` in all of the classes don't have the `CamelCase` also. We usualy have the first letter capitalised only for components. Again **consistency** !